core/src/rpc.rs

```diff
@@ -3107,6 +3218,20 @@ pub mod rpc_full {
             meta.get_stake_activation(&pubkey, config)
         }
 
+        fn get_inflation_reward(
```

Comment:
What do you think about just supporting a single epoch here?  Yes this means that the client would need to make multiple calls to fetch older epochs but I think that's better than a single very large RPC call

Reply:
Ah, maybe I misinterpreted original reason for adding this. Sure, that'll be a quick change!

Reply:
+1 for the single epoch. Default to most recent if epoch parameter not included?

Reply:
Good call. I'll default to the most recent.